crates/holochain_sqlite/src/sql/conductor/select_valid_cap_grant_for_cap_secret.sql

```diff
@@ -2,17 +2,20 @@ SELECT
   Entry.blob
 FROM
   Entry
+  INNER JOIN Action ON Action.author = ?2
+  AND Action.entry_hash = Entry.hash
```

Comment:
```suggestion
  INNER JOIN Action ON Action.entry_hash = Entry.hash
```

I believe `Action.author = ?2` should be part of the outer WHERE clause since it's not joining two tables together

Reply:
In that case I prefer to write all filters in the where clause
```sql
SELECT
  Entry.blob
FROM
  Entry, Action
WHERE
  Entry.access_type = ?1
  AND Action.author = ?2
  AND Action.entry_hash = Entry.hash
  AND (
    -- cap grant must not have been updated or deleted
    SELECT
      COUNT(UpdateActions.hash)
    FROM
      Action as UpdateActions
    WHERE
      UpdateActions.author = ?2
      AND (
        UpdateActions.original_entry_hash = Entry.hash
        OR UpdateActions.deletes_entry_hash = Entry.hash
      )
  ) = 0;
```

I find that easier to read.

Reply:
Actually it looks like [there is no difference](https://stackoverflow.com/questions/44917/explicit-vs-implicit-sql-joins)